addons/medical_vitals/functions/fnc_handleUnitVitals.sqf

```diff
@@ -98,15 +98,28 @@ switch (true) do {
         TRACE_2("heartRate Fatal",_unit,_heartRate);
         [QEGVAR(medical,FatalVitals), _unit] call CBA_fnc_localEvent;
     };
-    case (_bloodPressureH < 50): {
-        TRACE_2("bloodPressureH Fatal",_unit,_bloodPressureH);
+    case (_bloodPressureH < 50 && {_bloodPressureL < 40} && {_heartRate < 40}): {
+        TRACE_4("bloodPressure (H & L) + heartRate Fatal",_unit,_bloodPressureH,_bloodPressureL,_heartRate);
         [QEGVAR(medical,FatalVitals), _unit] call CBA_fnc_localEvent;
     };
-    case (_bloodPressureL < 40): {
-        [QEGVAR(medical,CriticalVitals), _unit] call CBA_fnc_localEvent;
+    case (_bloodPressureL => 190) {
+        TRACE_2("bloodPressure L above limits",_unit,_bloodPressureL);
+        [QEGVAR(medical,FatalVitals), _unit] call CBA_fnc_localEvent;
     };
-    case (_heartRate < 30): {
-        [QEGVAR(medical,CriticalVitals), _unit] call CBA_fnc_localEvent;
+    case (_heartRate < 30): {  // With a heart rate below 30 but bigger than 20 there is a chance to enter the cardiac arrest state
+        private _lastCheck = _unit getVariable [QGVAR(lastCheckCriticalHeartRate), CBA_missionTime];
+        private _enterCardiacArrest = false;
+        if (_lastCheck <= CBA_missionTime) then {
```

Comment:
Minor readability thing, but I'd probably change this to `_nextCheck` and swap the inequality comparison around to `CBA_missionTime > _nextCheck`. Just makes the intention of the code more clear (to me at least).